src/App.css

```diff
@@ -1,23 +1,26 @@
 body {
-  background: url("../public/wall.jpg");
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  width: 100%;
+
+background: url(../public/wallpaper.jpg)  no-repeat  center center;
+background-size: cover;
+display: flex;
+flex-direction: row;
+justify-content: center;
+
+min-height: 100vh;
 }
 
-.container {
-  background: #cec6c8;
-  opacity: 50%;
+.container { 
+  color: #27221e;
+  background: #bcb1b4;
+  opacity: 95%;
```

Comment:
use integer values like 0.95 instead of percentage

Reply:
done